charts/latest/blobfuse-csi-driver/templates/csi-blobfuse-driver.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: blobfuse.csi.azure.com
+spec:
+  attachRequired: true
+  podInfoOnMount: true
+  volumeLifecycleModes:  # added in Kubernetes 1.16
```

Comment:
we still need to support 1.14 for a period of time
/hold

Reply:
removed it.